### PR TITLE
SUP-4591 - Deprecation of clear_file_sync_locks [st0267]

### PR DESCRIPTION
**Pull request by Aaronoftheages**
Add deprecation notice to the task file and updated readme to include alternative steps

Screenshot 2023-11-06 at 16 31 31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The following tasks are no longer being developed and will be deprecated in a fu
 
 | Task Name | Alternative |
 |-----------|-------------|
-| st0317a_clean_cert | User [certificate clean](https://www.puppet.com/docs/puppet/7/server/http_certificate_clean) API to remove certifications |
+| st0317a_clean_cert | Use [certificate clean](https://www.puppet.com/docs/puppet/7/server/http_certificate_clean) API to remove certifications |
```
Comment (m0dular) [inline README.md:40]: Is this a copy-paste mistake?  This api won't clear any file sync locks.

**Review by m0dular — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Aaronoftheages) [reply README.md:40]: Yeah that was a mistake, I'll remove it

Comment (Aaronoftheages) [reply README.md:40]: I noticed the spell check and realized it needed corrected

File: README.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ The following tasks are no longer being developed and will be deprecated in a fu
 | Task Name | Alternative |
 |-----------|-------------|
 | st0236_set_cache_paths_to_default | Use [puppet conf](https://forge.puppet.com/modules/puppetlabs/puppet_conf/readme) |
+| st0267_clear_file_sync_locks | statedir can be changed [manually](https://www.puppet.com/docs/puppet/7/configuration.html#statedir) |
```
Comment (m0dular) [inline README.md:41]: What does changing the `statedir` have to do with this task?  This is about removing stale `index.lock` files that prevent file sync from working.  For an alternative, we need to either list the manual steps from the [kb article](https://perforce1--uat.sandbox.my.site.com/Community2/s/article/360003883933) or link to it.  That still references the task, so we also need to change the article before linking to it.

**Review by m0dular — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Aaronoftheages) [reply README.md:41]: I've put the wrong resolution message in from the documentation - I'll correct that now

Comment (Aaronoftheages) [reply README.md]: https://perforce.atlassian.net/browse/SUP-4623

resolved, the KB article will need updated - Ticket created

**Review by m0dular — APPROVED** [review]

**Pull request merged**